sdk/index.md

```diff
@@ -30,7 +30,7 @@ An enclave application partitions itself into two components (1) an untrusted co
    <div class="column3"  ><img src="../assets/images/Standardized.png" width="50" /><h4>Standardized</h4><p>Remove hardware vendor specific signing and verification requirements</p></div>
   </div>
 <div class="row3">
-   <div class="column3"><img src="../assets/images/Multiplatform.png" width="50" /><h4>Multi-platform</h4><p>Design with all software platforms, Windows and Linux, in mind</p></div>
+   <div class="column3"><img src="../assets/images/Multiplatform.png" width="50" /><h4>Multi-platform</h4><p>Design with all SGX/enclave supported software platforms, Windows and Linux, in mind</p></div>
```

Comment:
I don't really think this is what we want. The goal is to support any platform with TEE support. How about

> "Design with all TEE supported software platforms in mind"

Today that's Windows and Linux, tomorrow who knows. This is just saying what the protect goals are, we list what OS's we officially support elsewhere.